requirements.txt

```diff
@@ -0,0 +1,2 @@
+BeautifulSoup==3.2.1
+wsgiref==0.1.2
```

Comment:
Cool! Thanks. Could you please notify with a message when you have updated the pull request. I'm not sure about how I get notified on Github. :)